assets/msalv2.js

```diff
@@ -173,11 +173,51 @@ var aadOauth = (function () {
       .catch(onError);
   }
 
-  function getAccessToken() {
+  async function getAccessToken() {
+    const account = getAccount();
+
+    if (account !== null && authResult === null) {
+      try {
+        // Silent acquisition only works if we the access token is either
```

Comment:
Remove " we" from comment

Reply:
This is copy & pasted code. Resolved in the fix addressing the other comment.

https://github.com/Earlybyte/aad_oauth/pull/215/commits/b5baf8080449f5786ab29f7f1aed952e8a9b6d19